### PR TITLE
Cumulative update of GH flow

### PR DESCRIPTION
**Pull request by cstamas**
Instead to do this one-by-one as pushed by dependendabot...

### CODE REVIEW

**Comment by cstamas** [comment]
related
https://github.com/graalvm/setup-graalvm?tab=readme-ov-file#migrating-from-graalvm-223-or-earlier-to-the-new-graalvm-for-jdk-17-and-later

File: .github/workflows/early-access.yaml [hunk 1]
```diff
@@ -52,12 +57,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-22.04, macOS-11, windows-2019 ]
+        os: [ ubuntu, macos, windows ]
```
Comment (slawekjaranowski) [inline .github/workflows/early-access.yaml:60]: There are not runners like `ubuntu`
There are `ubuntu-latest` and so on ...

https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners

**Comment by cstamas** [comment]
@gzm55 help! :smile:

**Comment by gzm55** [comment]
> @gzm55 help! 😄

@cstamas try append this line to client/src/main/resources/glibc/glibc.redef

```
pthread_mutex_trylock pthread_mutex_trylock@GLIBC_2.2.5
```

**Comment by gzm55** [comment]
> @gzm55 Thanks a lot! It seems it worked, ubuntu job is green...

yeah, new graalvm introduce new api dependency.

**Review by gnodet — APPROVED** [review]

**Pull request merged**